kover-gradle-plugin/docs/index.md

```diff
@@ -36,11 +42,713 @@ Gradle plugin that measures the coverage for tests running on the JVM and genera
+- To print coverage to the log, call `:koverLog` Gradle task
+
+If a module with the path `:foo` was selected as the merging module, then you will need to call commands like `:foo:koverXmlReport`, etc.
+
+It is recommended to use only full path commands since short names like `koverXmlReport` will run all tests in all projects, even if they are not specified as `kover` dependencies.
+
+Running one of these tasks automatically triggers the launch of all test tasks from merging module and the modules specified in it as kover dependencies.
+
+To exclude the automatic launch of a certain task and never take into account the coverage of this task, you should additionally [configure the corresponded project](#exclusion-of-test-tasks).
+
+#### Example
+The full example for multi-module Kotlin JVM project can be found [here](https://github.com/Kotlin/kotlinx-kover/tree/main/kover-gradle-plugin/examples/jvm/merged).
+
+#### Adding verification rules
+
+To add verification rules, in the build script file of the merging module (and only in it) it is necessary to specify the following:
+```kotlin
+kover {
+    reports {
+        verify {
+            // add new verification rule
+            rule {
+                // specify coverage bounds for the rule
+            } 
+        }
+    }
+}
+```
+You can read more about creating verification rules in the [corresponding section](#verification).
+
+#### Configuring Reports
+The configuration of the report generation tasks takes place in a block for aggregated tasks in the build script file of the merging module (and only in it). 
+
+```kotlin
+kover {
+    reports {
+        total {
+            // configuring report tasks
+        }
+    }
+}
+```
+
+Available configuration options are listed in a [separate section](#configuring-report-tasks).
+
+----
+
+### Multi-module Kotlin multiplatform project
+
+Kover supports coverage measurement only for JVM targets.
+Source code specified not in the common and not JVM source sets are ignored.
+As well as coverage from non-JVM tests is also not taken into account.
+
+Otherwise, the use of the plugin is identical to the use in the [multi-module Kotlin JVM project](#multi-module-kotlin-jvm-project).
+
+----
+
+### Kotlin single-module Android project
+#### Application of Kover plugin
+Add the following to build file in single `app` module of your Gradle build:
+```kotlin
+plugins {
+     id("org.jetbrains.kotlinx.kover") version "0.7.6"
+}
+```
+
+For more information about application of the plugin, refer to the [relevant section](#apply-kover-gradle-plugin-in-project)
+
+After applying Kover Gradle plugin, the classes are modified (instrumented) when loaded into the JVM during local unit test runs,
+so number of line hits can be counted. This may impact your testing experience.
+At the moment, the connected tests are not being instrumented.
```

Comment:
Official documentation calls them instrumented tests, so we are in a kinda dumb situation: 
> At the moment, the instrumented (on-device) tests are not being instrumented.

Maybe rephrase this as `At the moment, the instrumented (on-device) tests are not being modified by Kover and coverage for them can not be collected.`?